Add Meson Build Support (#1021)
rauc/rauc: README.rst

---------------------

.. note:: RAUC is intended to be built both as a host tool as well as a target

>    tool (service). Therefore it is fully prepared for meson.

Reviewer: ```suggestion
   tool (service). Therefore it is fully prepared for cross-compilation with meson.
```

::

    git clone https://github.com/rauc/rauc
    cd rauc
    meson setup build
    ninja -C build

> .. note:: At the moment also `automake cross-compilation

Reviewer: ```suggestion
.. note:: At the moment, also `automake cross-compilation
```

   <https://www.gnu.org/software/automake/manual/html_node/Cross_002dCompilation.html>`_

>    is still supported, but the plan is to deprecate this in the new future

Reviewer: ```suggestion
   is still supported, but the plan is to deprecate this soon.
```
